data/images/tiles.strf

```diff
@@ -5117,7 +5145,7 @@
     (width 8)
     (height 3)
     (ids
-      3398 3399 3400 3401 3402 3403 3404 3405
+         0    0 3401    0    0    0    0    0

```

Comment:
Is 3401 intended to be 1 to the left here?

Those tiles should also be marked as 0 in `attributes`.